bulbs/ads/static/js/mobile-ad-placer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var _ = require('lodash/lodash');
+var adHtml = '<div class="dfp dfp-slot-inread" data-ad-unit="inread"></div>';
+
+
+var MobileAdPlacer = {
+  placeAds: function () {
+    var body = $('body');
+    var paragraphs = body.find('p');
+    var wordCount = 0;
+    var adsPlaced = 0;
+
+    for (var i = 0; i < paragraphs.length; i++) {
```

Comment:
Use a `forEach` loop or one of the other [built in `Array` functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/prototype#Methods) here.